proto/cosmos/base/store/snapshots/v1/snapshot.proto

```diff
@@ -1,5 +1,5 @@
 syntax = "proto3";
-package cosmos.base.store.snapshots.v1beta1;
+package cosmos.base.store.snapshots.v1;
```

Comment:
```suggestion
package cosmos.store.snapshots.v1;
```

Small proposal for a shorter name. 👍 / 👎 ?

Reply:
cc @tac0turtle

Reply:
it looks nice but it fails proto lint since the linter wants the path to the be the package

Reply:
Right, we would just need to move the *.proto files to a folder higher.

Reply:
yea open to that

Reply:
Done in https://github.com/cosmos/cosmos-sdk/pull/14597/commits/9d3b6204ce4569a2b0d689fcd23fa7b949d6002d